### PR TITLE
RHBPMS-5016: Task selection returns "Could not find task instance wit…

### PR DESCRIPTION
**Pull request by nmirasch**
…h id" when task belongs to aborted process instance
Display readonly details for tasks with status exited and completed

### CODE REVIEW

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenterTest.java [hunk 1]
```diff
@@ -91,25 +95,20 @@
 
     @Mock
     DataSetQueryHelper dataSetQueryHelperDomainSpecific;
-
+    @Mock
+    PlaceManager placeManager;
     @Mock
     private TaskListViewImpl viewMock;
-
```
Comment (cristianonicolai) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenterTest.java]: not really sure we need to remove lines between the variables :)

**Comment by cristianonicolai** [comment]
just a minor comment, other than that looks good to me.

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenterTest.java [hunk 2]
```diff
@@ -565,4 +564,62 @@ public void testStatusSettingsColumns(){
...
+        when(placeManager.getStatus(any(DefaultPlaceRequest.class))).thenReturn(PlaceStatus.CLOSE);
+
+        getPresenter().selectTask(taskSummary,
+                                  closed);
+
+        verify(placeManager).goTo(TaskDetailsMultiPresenter.SCREEN_ID);
+        final ArgumentCaptor<TaskSelectionEvent> captor = ArgumentCaptor.forClass(TaskSelectionEvent.class);
+        verify(taskSelected).fire(captor.capture());
+        assertTrue(captor.getValue().isForLog());
+    }
```
Comment (bsremac) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenterTest.java]: Maybe we don't need to tests both read-only statuses separately. I think testing only one won't reduce test coverage in any way.

**Comment by nmirasch** [comment]
@cristianonicolai minnor comment fixed. @bsremac removed one of read-only tests.
Could you take a look? Thanks!

**Pull request merged**